subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_iso.c

```diff
@@ -782,14 +874,72 @@ static void next_chan_calc(struct lll_adv_iso *lll, uint16_t event_counter,
 		data_chan_id = lll_chan_id(access_addr);
 
 		/* Calculate the radio channel to use for next BIS */
-		lll->next_chan_use = lll_chan_iso_event(event_counter,
-						data_chan_id,
-						lll->data_chan_map,
-						lll->data_chan_count,
-						&lll->data_chan_prn_s,
-						&lll->data_chan_remap_idx);
+		lll->next_chan_use =
+			lll_chan_iso_event(event_counter,
+					   data_chan_id,
+					   lll->data_chan_map,
+					   lll->data_chan_count,
+					   &lll->data_chan.prn_s,
+					   &lll->data_chan.remap_idx);
+	}
+}
+#endif /* CONFIG_BT_CTLR_ADV_ISO_SEQUENTIAL */
+
+#if defined(CONFIG_BT_CTLR_ADV_ISO_INTERLEAVED)
+static void next_chan_calc_int(struct lll_adv_iso *lll, uint16_t event_counter)
```

Comment:
Did you consider having this function return `uint8_t` rather than modifying the variable in `lll`? 

i.e. do `lll->next_chan_use = next_chan_calc_int(lll, event_count);`